src/main.rs

```diff
@@ -20,6 +20,10 @@
 
 use color_eyre::eyre;
 
+/// Global allocator
+#[global_allocator]
+pub static ALLOC: tikv_jemallocator::Jemalloc = tikv_jemallocator::Jemalloc;
```

Comment:
maybe we should add a note, that changing to another allocator it will require changing `memory_stats::MemoryAllocationTracker`